oval_graph/json_to_html.py

```diff
@@ -80,21 +82,16 @@ def _get_wanted_not_selected_rules(self):
         return []
 
     def prepare_data(self, rules):
-        try:
-            out = []
-            for rule in rules["rules"]:
-                self.oval_tree = self.load_json_to_oval_tree(rule)
-                oval_tree_dict = self.create_dict_of_oval_node(self.oval_tree)
-                src = self.get_save_src(rule.replace('graph-of-', '') + "-")
-                self.save_html_report(oval_tree_dict, src)
-                self.open_web_browser(src)
-                print('Rule "{}" done!'.format(rule))
-                out.append(src)
-            return out
-        except Exception as error:
-            raise ValueError(
-                'Rule: "{}" Error: "{}"'.format(
-                    self.source_filename, error))
+        out = []
+        for rule in rules["rules"]:
+            self.oval_tree = self.load_json_to_oval_tree(rule)
+            oval_tree_dict = self.create_dict_of_oval_node(self.oval_tree)
+            src = self.get_save_src(rule.replace('graph-of-', '') + "-")
+            self.save_html_report(oval_tree_dict, src)
+            self.open_web_browser(src)
+            print('Rule "{}" done!'.format(rule))
+            out.append(src)
+        return out
```

Comment:
This code looks similar to the code in `prepare_data` in `arf_to_html`. Would it be feasible to unify it?

Reply:
OK, I will remove this duplicate code in another PR.